extensions/markdown-language-features/src/commands/showPreview.ts

```diff
@@ -68,6 +68,21 @@ export class ShowPreviewCommand implements Command {
 	}
 }
 
+export class CopyImageCommand implements Command {
+	public readonly id = 'markdown.copyImage';
+
+	public constructor(
+		private readonly _webviewManager: MarkdownPreviewManager,
+		_telemetryReporter: TelemetryReporter
+	) { }
+
+	public execute(args: { id: string }) {
+		if (this._webviewManager._activePreview) {
+			this._webviewManager._activePreview.copyImage(args.id);
```

Comment:
You can use optional changing here to shorten the code: 

```
this._webviewManager._activePreview?.copyImage(args.id);
```

This will only call `copyImage` if active preview is defined